### PR TITLE
Facade

### PR DESCRIPTION
**Pull request by hayatariq**
I implemented the facade! and also delete panel ui in profileresponsebox class. I did something to the setresponse method, instead of working with loggedinuser it is now working with loggedinuserid, it helps me out. please lmk if it works for you and if not i'll try to work with the user objects.

### CODE REVIEW

**Comment by NimratGrewal** [comment]
hi haya, just had a question: is the delete panel just adding the delete button per response box?

**Comment by hayatariq** [comment]
yes ma'am , is there an issue with the code?

**Comment by hayatariq** [comment]
all it does is hold the button

**Comment by NimratGrewal** [comment]
> yes ma'am , is there an issue with the code?

nope! just wanted to clarify! ty

**Review by momoasalu — COMMENTED** [review]
You can just create another method that returns the ID! You don’t need to change the other one as some classes already use it as it is.

File: src/use_case/toFeed/FeedInteractor.java [hunk 1]
```diff
@@ -1,34 +1,30 @@
...
-import use_case.toProfile.UserProfileDataAccessInterface;
 
 import javax.swing.*;
 import java.time.LocalDate;
 import java.util.*;
 
 public class FeedInteractor implements FeedInputBoundary {
-    private final UserProfileDataAccessInterface userDataAccessObject;
-    private final FeedDataAccessInterface feedDataAccessObject;
+    private final DataAccessObjectFacade dataAccessObjectFacade;
```
Comment (momoasalu) [inline src/use_case/toFeed/FeedInteractor.java:13]: You still should be using an interface here. Just have the facade implement it.

**Comment by hayatariq** [comment]
i just updated my use cases after my initial testing!

**Review by chowdberry — APPROVED** [review]
In general, the implementation seems to be well done. Addition of extra tests would be helpful in identifying if there are any major mistakes that need to be rectified. Documentation for areas such as FeedView are very good, and it would be good if helpful comments can be added in other areas such as FeedResponseBox as well. Otherwise, great job!

**Pull request merged**